### PR TITLE
Update readme and doc image

### PR DESCRIPTION
**Pull request by stevejgordon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,20 +189,20 @@ builder.Services.AddOpenTelemetry()
 
 ### Manual Instrumentation usage
 
-In environments where no `IServiceCollection` is naturally available you may manually start instrumenting by creating an instance of `ElasticOpenTelemetryBuilder`. 
+In environments where an `IServiceCollection` is unavailable you may manually start instrumenting by creating
```
Comment (Mpdreamz) [inline README.md:192]: Just curious what's introducing this line breaks? It might be our line length in `.editorconfig`?

**Review by Mpdreamz — CHANGES_REQUESTED** [review]
If we link the images through their full path on `github.com` they should also display correctly on nuget see: 

https://learn.microsoft.com/en-us/nuget/nuget-org/package-readme-on-nuget-org#allowed-domains-for-images-and-badges

https://www.nuget.org/packages/Elastic.OpenTelemetry/1.0.0-alpha.1

File: README.md (hunk 1 above)
Comment (stevejgordon) [reply README.md:192]: Just me manually tweaking them. Istvan, in the past, told me there was a preferred width, and I've got into the habit of roughly sticking to that. It happens to be about the width I have when a preview is open side-by-side.

**Comment by stevejgordon** [comment]
> If we link the images through their full path on `github.com` they should also display correctly on nuget see:
> 
> https://learn.microsoft.com/en-us/nuget/nuget-org/package-readme-on-nuget-org#allowed-domains-for-images-and-badges
> 
> https://www.nuget.org/packages/Elastic.OpenTelemetry/1.0.0-alpha.1

I can update although, looking at that list is it only `github.com/.../workflows/.../badge.svg` that they allow?

**Comment by Mpdreamz** [comment]
The images can be referenced through: 

raw.github.com
raw.githubusercontent.com

Which is allow listed 🎉

File: README.md (hunk 1 above)
Comment (Mpdreamz) [reply README.md:192]: Cool we should add this explicitly to `.editorconfig` at some point.

**Review by Mpdreamz — APPROVED** [review]

**Pull request merged**